app/models/grouping.rb

```diff
@@ -421,6 +421,7 @@ def marking_completed?
   def remove_member(mbr_id)
     member = student_memberships.find(mbr_id)
     if member
+      $stderr.puts "In grouping"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.